<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inputs_example</class>
 <widget class="QWidget" name="Inputs_example">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">Inputs_example {background-color: white; }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="LineEdit" name="cross2" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="LineEdit" name="widget_4" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="LineEdit" name="widget" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="LineEdit" name="cross1" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="LineEdit" name="widget_2" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="LineEdit" name="widget_3" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>кросс-валидация, поля должны бать разными</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QWidget</extends>
   <header>lineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
